build: support `--skip-unused-stages` for multi-stage builds. (#4249)
containers/buildah: docs/buildah-build.1.md

Sign the built image using the GPG key that matches the specified fingerprint.

**--skip-unused-stages** *bool-value*

> Allows buildah to skip stages in a multi-stage builds which are marked as unused (i.e
> stage which don't contribute anything to the target stage directly or indirectly). (Default is `true`).

Reviewer: ```suggestion
Skip stages in multi-stage builds which don't affect the target stage. (Default is `true`).
```
